src/appendices/hed.md

```diff
@@ -106,18 +106,20 @@ The `duration` column is also annotated as a value column.
 
 Dedicated HED tools MUST assemble the HED annotation for each event (row) by concatenating the
 annotations for each column, along with the annotation contained directly in a `HED` column
-of that row as described in the next section.
+of that row, as described in the next section.
 
-Example: The fully assembled annotation for the first event in the above
+**Example:** The fully assembled annotation for the first event in the above
 `events.tsv` file with onset `1.2` (the first row) is:
 
 ```Text
-Duration/0.6 s, Sensory-event, Visual-presentation, (Square, Blue),
-(Delay/1.435 ms, Agent-action, (Experiment-participant, (Press, Mouse-button))),
+(Duration/0.6 s, Sensory-event, Visual-presentation,
```

Comment:
This is not correct without the curly brace notation, which is not yet supported by the HED javascript validator used in bids.  Until that all goes through, the annotation for the presentation cannot be enclosed in parentheses:

```code
Duration/0.6 s, Sensory-event, Visual-presentation, 
((Square, Red), (Computer-screen, Center-of)),
```

@effigies can you correct.  Tal will not be available for a while.